lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -3366,6 +3366,14 @@ public function supportsColumnCollation()
         return false;
     }
 
+    /**
+     * Does this platform support column reordering in alter table?
+     */
+    public function supportsUpdateColumnOrder(): bool
```

Comment:
or `supportsColumnRerder`?

Reply:
How is it supposed to be used? The approach of adding `supports*` methods implies that the actual platform-specific logic will be implemented outside of the platform in the calling code. It might be better to define the API in terms of actions, e.g. `reorderColumns()`.